concourse/tasks/build-container-image.yaml

```diff
@@ -0,0 +1,19 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: gcr.io/kaniko-project/executor
+    tag: debug
+
+inputs:
+- name: guest-test-infra
+- name: credentials
+
+run:
+  path: sh
+  args:
+  - -exc
+  - |
+    export GOOGLE_APPLICATION_CREDENTIALS=$PWD/credentials/credentials.json
+    executor --dockerfile=((dockerfile)) --context=./guest-test-infra --destination=((destination))
```

Comment:
this is a generic task 'build-container-image' it should take the context as an argument, not assume it

Reply:
Will use var for context